text/0000-no-more-extern-crate.md

```diff
@@ -70,29 +85,21 @@ For example, `cargo build`-ing a crate `my_crate` with a dependency on `rand`
 results in a call to rustc that looks something like
 `rustc --crate-name mycrate src/main.rs --extern rand=/path/to/librand.rlib ...`.
 
-When an external crate is specified this way,
-the crate will automatically brought into scope as if an
-`extern crate name_of_crate;`
-declaration had been added to the current crate root.
-This behavior won't occur when including a library using the `-l`
+When an external crate is specified this way, it will be automatically
+available to any module in the current crate through `use` statements or
+absolute paths (e.g. `::rand::random()`). It will _not_ be automatically
+imported at root level as happens with current `extern crate`.
```

Comment:
I explained this previously.

First people complain that local/global distinction in non-use/use paths is confusing and want to get rid of it, and then suggest something twice as bad and inconsistent and breaking the language logic.
Note that the local/global distinction had good intentions behind it too, e.g. avoiding breaking `use std::xyz` imports when they are moved into inner modules.

In addition, the primary purpose of this RFC as originally written was removing the duplication between `Cargo.toml` and `lib.rs`, "removing the need for editing Rust source files" citing @withoutboats .
With the recent update the duplication is kept in place, just reworded into more ambiguous and confusing form.